generate_and_send_a_creative_email_using_openai_and_stability_ai/Ballerina.toml

```diff
@@ -0,0 +1,8 @@
+[package]
+org = "ai_samples"
+name = "generate_images_using_stabilityai"
```

Comment:
shouldn't this be the same name as the sample?

Reply:
Addressed in https://github.com/ballerina-guides/ai-samples/pull/67/commits/ee50938cc116cc77b3afab1d411343826a263002.